backend/src/election/mod.rs

```diff
@@ -90,3 +91,43 @@ pub async fn election_status(
     let statuses = polling_station_repo.status(id).await?;
     Ok(Json(ElectionStatusResponse { statuses }))
 }
+
+/// Download a generated PDF
+#[utoipa::path(
+    get,
+    path = "/api/elections/{election_id}/download_pdf",
```

Comment:
I find `download_pdf` a little vague as it describes the format, not the data.

We can change it to `download_results`? ("Certified Election Results" is the translation of "Proces Verbaal" we have in our "woordenlijst". I also suppose they are not actually certified, until the documents are signed?)

Am also fine with changing the endpoint, once we generate the pdf with the results, since for the scope of this PR, `download_pdf` arguably is a better name than `download_results`.

Reply:
We haven't decided yet on the exact API. That will become clear once we develop more of the epic. I've changed anyway in [2fee568](https://github.com/kiesraad/abacus/pull/262/commits/2fee568fb06d7df540929b85831709e53a770496), since it's indeed probably better